eva/eva.yml

```diff
@@ -17,7 +17,7 @@ storage:
   engine: "eva.storage.petastorm_storage_engine.PetastormStorageEngine"
   video_engine: "eva.storage.opencv_storage_engine.OpenCVStorageEngine"
   video_engine_version: 0
-  path_prefix: "/tmp"
+  upload_dir: ""
```

Comment:
We can avoid exposing upload_dir and always assume `tmp` implicit to our system.

Reply:
Internally, we refer to it as the Upload directory so the logic still remains the same